source/globalCommands.py

```diff
@@ -1786,7 +1785,7 @@ def script_review_copy(self, gesture):
 				pass
 			try:
 				copyMarker.updateSelection()
-				if isinstance(pos.obj, editableText.EditableTextWithoutAutoSelectDetection):
+				if hasattr(pos.obj, "waitForAndReportSelectionChange"):

```

Comment:
Ah yes, I forgot about case 1. I think the renaming it as you suggest is the most pragmatic solution.